src/borg/archiver.py

```diff
@@ -2929,7 +2922,8 @@ def process_epilog(epilog):
         Note that all paths in an archive are relative, therefore absolute patterns/paths
         will *not* match (--exclude, --exclude-from, PATHs).
 
-        --compression: all chunks seen will be stored using the given method.
+        --recompress: all chunks seen will be recompressed using the --compression
+        specified.
```

Comment:
unrelated to your change, but I just noticed that this is not quite true:
they are not recompressed if they already have that compression, except with --always-recompress.